style.css

```diff
@@ -0,0 +1,89 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  left: 0;
+  top: 0;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.main-header {
+  display: flex;
+  justify-content: right;
+  margin-top: 63px;
+  flex-wrap: wrap;
+}
+
```

Comment:
- [ ] As requested by the previous reviewer please 🙏 make sure the `margin-top` of you` header section` is such that it looks as close to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=gHYreHPQqfVOzLLb-0) as possible. Currently your header section is too far off from the top I suggest reducing the `margin-top` or use `padding` instead of margin

Your Design:

<img width="376" alt="Screenshot 2023-03-23 at 11 20 54 AM" src="https://user-images.githubusercontent.com/44798044/227121671-2a177a97-efb7-43b5-a961-6b0fe2781b93.png">


Figma Design:

<img width="472" alt="Screenshot 2023-03-23 at 11 36 10 AM" src="https://user-images.githubusercontent.com/44798044/227122996-6d541f59-61e6-4d60-8dbf-340f78a5a831.png">